docs/index.asciidoc

```diff
@@ -28,6 +28,12 @@ Notes:
 * The `command` field of this event will be the command run.
 * The `message` field of this event will be the entire stdout of the command.
 
+===== IMPORTANT
+
+The exec input ultimately uses fork to spawn a child process.
```

Comment:
Use backticks for `fork`?  Otherwise LGTM.

Reply:
```suggestion
The exec input ultimately uses `fork` to spawn a child process.
```